cms/server/admin/static/aws_style.css

```
@@ -2,7 +2,29 @@
+
+.footer {
+    all: unset !important;
+}
+
 :root {
+    --bulma-primary-h: 175deg;
+    --bulma-primary-s: 77%;
+    --bulma-primary-l: 26%;
+
+    /* Font family - Inter */
+    --bulma-family-primary: 'Inter', system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
```

Stylelint (17.2.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)